src/components/Modal/VaultStrategyModal/VaultStrategyRightPanel.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react"
```

Comment:
I think this is getting a little convoluted. It's harder to work on the code when we have it split into *this many* files. The strategy modals are not complex enough to warrant splitting the code down into 'left panel' and 'right panel'.

We also don't need `VaultStrategyModal` at all; it's an added layer of abstraction without much purpose since each of the product-specific modals are used in their own unique place:

- `ConcentratorStrategyModal` can be used in `concentrators.tsx` 
- `CompounderStrategyModal` can be used in `yield/index.tsx` 
- `ManagedStrategyModal` can be used in `managed-vaults.tsx`

Reply:
I see.